docker/dockerfiles/bedrock_base

```diff
@@ -11,11 +11,10 @@ CMD ["./docker/run-prod.sh"]
 RUN adduser --uid 1000 --disabled-password --gecos '' --no-create-home webdev
 
 RUN apt-get update && \
-    apt-get install -y --no-install-recommends python2.7 libpython2.7 gettext build-essential libxml2-dev libxslt1.1 libxslt1-dev python-dev python-pip nodejs npm git postgresql-client libpq-dev
-RUN update-alternatives --install /usr/bin/node node /usr/bin/nodejs 10
-
-COPY ./bin/pipstrap.py /app/bin/pipstrap.py
-RUN ./bin/pipstrap.py
+    apt-get install -y --no-install-recommends \
+                    gettext build-essential python python-dev python-setuptools \
+                    libxml2-dev libxslt1.1 libxslt1-dev \
+                    nodejs-legacy npm git postgresql-client libpq-dev
```

Comment:
The postgresql-client install will not work until the latest base image includes https://github.com/mozmar/docker-base/pull/1, and as I've mentioned before I don't think it really belongs in the base image anyway-- we can optionally include it in the dev image later.